frontend/src/pages/ConversationPage/ConversationPage.tsx

```diff
@@ -4,15 +4,26 @@ import ConversationBox from "./components/ConversationBox"
 import NavBar from "../../common/components/NavBar/NavBar"
 import { useContext, useState } from "react"
 import { AuthContext } from "../../common/providers/AuthProvider"
+import { ConversationService } from "./services/ConversationService"
 
 export default function ConversationPage() {
   const [isShowReport, setIsShowReport] = useState(false)
   const [selectedConversationId, setSelectedConversationId] = useState<string | null>(null)
   const { userId, userType, username } = useContext(AuthContext)
   const [isSystemReport, setIsSystemReport] = useState(false)
+  const [selectedUserType, setSelectedUserType] = useState<"CUSTOMER" | "FORTUNE_TELLER">(
+    "FORTUNE_TELLER"
+  )
```

Comment:
We have UserType type in frontend duai na, but it's okay in this case.  e.g. Exclude<UserType | "ADMIN">

คือถ้าอยากใช้แค่ customer, fortune-teller บางทีประกาศ type ไป ถึงแม่นกว่าแต่ก็อ่านยากกว่า ก็เลยแบบ ประกาศงี้ไปเลยก็รับได้ แค่บอกเฉย ๆ ว่ามี UserType ด้วย